packages/docusaurus-theme-search/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@markprompt/docusaurus-theme-search",
```

Comment:
Would this be enough? Or do you suspect we might be releasing other Docusaurus plugins in the future? 

```suggestion
  "name": "@markprompt/docusaurus",
```

Reply:
Ah no I see the naming conventions now at https://docusaurus.io/docs/using-plugins#module-shorthands. This is as it should be 👌

Reply:
In fact, I started using the name you proposed.